File: project_java/jdbc_add_course.php
<html>

<body>
    <h3>Enter information about a Course to add to the database:</h3>

    <form action="jdbc_add_course.php" method="post">
        DeptCode: <input type="text" name="code"><br>
        CourseNum: <input type="text" name="num"><br>
        Type: <input type="text" name="title"><br>
        City: <input type="text" name="credit"><br>
        <input name="submit" type="submit">
    </form>
    <br><br>

</body>

</html>

<?php
if (isset($_POST['submit'])) {
    //Course: DeptCode, CourseNum, Title, CreditHours
    // replace ' ' with '\ ' in the strings so they are treated as single command line args
    $code = escapeshellarg($_POST[id]);
    $num = escapeshellarg($_POST[name]);
    $title = escapeshellarg($_POST[major]);
    $credit = escapeshellarg($_POST[credit]);

    $command = 'java -cp .:mysql-connector-java-5.1.40-bin.jar jdbc_add_course ' . $code . ' ' . $num . ' ' . $title . ' ' . $credit;

    // remove dangerous characters from command to protect web server
    $command = escapeshellcmd($command);
    echo "<p>command: $command <p>";

    // run
    system($command);
}
?>

File: project_java/jdbc_add_course.java
import java.sql.*;

//Course: DeptCode, CourseNum, Title, CreditHours
public class jdbc_add_course {
   // The main program that inserts a restaurant
   public static void main(String[] args) throws SQLException {
      String Username = "gbglenn"; // Change to your own username
      String mysqlPassword = "EiGaix8o"; // Change to your own mysql Password

      // Connect to the database
      jdbc_db myDB = new jdbc_db();
      myDB.connect(Username, mysqlPassword);
      myDB.initDatabase();

      // For debugging purposes: Show the database before the insert
      StringBuilder builder = new StringBuilder();
      String query1 = "SELECT * from Course";
      builder.append("<br> Table Course before:" + myDB.query(query1) + "<br>");

      // Parse input string to get restauranrestaurant Name and Address
      String code = "STUDENT ID";
      String num = "STUDENT NAME";
      String title = "COURSE TITLE";
      String credit = "CREDIT HOURS";

      // Read command line arguments
      code = args[0];
      num = args[1];
      title = args[2];
      credit = args[3];

      // Insert the new Student
      //String input = "'" + code + "','" + num + "','" + title + "','" + credit + "'";
      String input = code + "," + num + ",'" + title + "'," + credit;
      builder.append("Command Executed: " + input);
      myDB.insert("Course (CourseNum, DeptCode, Title, CreditHours)", input);

      // For debugging purposes: Show the database after the insert
      builder.append("<br><br><br> Table Course after:" + myDB.query(query1));
      System.out.println(builder.toString());

      myDB.disConnect();
   }
}
